src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {Avatar} from '@momentum-ui/react';
-
+import Icon from '../generic/Icon/Icon';
 import webexComponentClasses from '../helpers';
 import {usePerson} from '../hooks';
 
+/**
+ * Renders the person image and status icon.
+ *
+ * @param {string} avatar  URL of the image.
+ * @param {string|null} status  Status of the person.
+ * @returns {object} JSX of the component
+ */
+function renderAvatarContent(avatar, status) {
```

Comment:
this function seems to complicate things a bit, try to see if the code is clearer without it